app/controllers/journeys/expenses/interest/InterestCYAController.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.journeys.expenses.interest
+
+import controllers.actions._
+import controllers.handleSubmitAnswersResult
+import models.common._
+import models.journeys.Journey.ExpensesInterest
+import models.journeys.expenses.interest.InterestJourneyAnswers
+import pages.expenses.interest.InterestCYAPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.SelfEmploymentService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import utils.Logging
+import viewmodels.checkAnswers.expenses.interest._
+import viewmodels.journeys.SummaryListCYA
+import views.html.standard.CheckYourAnswersView
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+
+class InterestCYAController @Inject() (override val messagesApi: MessagesApi,
```

Comment:
I think we are missing the title message for the cya page, as at the moment the title on the page just says `interestCya.title.individual`